components/ui/__tests__/badge.test.tsx

```diff
@@ -1,17 +1,11 @@
-import { render } from "@testing-library/react";
+import { render, screen } from "@testing-library/react";
 import { Badge } from "../badge/badge";
 
 describe("Badge", () => {
   it("renders correctly", () => {
-    const { container } = render(<Badge>test badge</Badge>);
+    render(<Badge>test badge</Badge>);
 
-    expect(container.firstChild).toMatchInlineSnapshot(`
-      <span
-        class="rounded-full bg-gray-100 text-gray-800 px-2.5 py-0.5 text-xs inline-flex items-center font-medium"
-      >
-        test badge
-      </span>
-    `);
+    expect(screen.getByText("test badge")).toBeInTheDocument();
```

Comment:
I think we can keep the `.toMatchInlineSnapshot()` usage here since we're actually testing the styling for this component and the snapshot result is very small and readable.